logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserModel.kt

```diff
@@ -6,11 +6,23 @@ import com.wire.kalium.logic.data.id.VALUE_DOMAIN_SEPARATOR
 import com.wire.kalium.logic.data.publicuser.model.OtherUser
 
 typealias UserId = QualifiedID
+typealias ConversationId = QualifiedID
```

Comment:
I think we already have in place this alias :D   so we can reuse it.
`com/wire/kalium/logic/data/id/QualifiedId.kt`

Reply:
😅